docs/components/skeleton.md

```diff
@@ -140,13 +140,13 @@ Defaults to `false`, you can use this prop to hide the progress indicator.
   <KSkeleton
     v-if="loadingNone"
     :delay-milliseconds="0"
-    hide-progress
     type="fullscreen-kong"
   />
+  <!-- Generic Spinner example -->
   <KSkeleton
     v-if="loading"
     :delay-milliseconds="0"
-    type="fullscreen-kong"
+    type="fullscreen-generic"
```

Comment:
Please update the code snippet too:

![image](https://user-images.githubusercontent.com/67973710/224424940-3d999845-402f-4644-ad1e-d28f281833eb.png)